src/scripts/overview_paper/C_impact_of_healthsystem_assumptions/scenario_impact_of_healthsystem.py

```diff
@@ -48,7 +48,8 @@ def log_configuration(self):
         }
 
     def modules(self):
-        return fullmodel(resourcefilepath=self.resources) + [ScenarioSwitcher(resourcefilepath=self.resources)]
+        return fullmodel(resourcefilepath=self.resources) + [
+            ImprovedHealthSystemAndCareSeekingScenarioSwitcher(resourcefilepath=self.resources)]
```

Comment:
It's because we don't  _need_ to have `ImprovedHealthSystemAndCareSeekingScenarioSwitcher` in the `full_model` (although it wouldn't do any harm if it was, to be honest!). So, as it's not, I have to concatenate list of modules to pass in at this point.

We could add 'ScenarioSwticher' to the full model, but I think it could add complexity to the full model that we don't need.

Reply:
I see. You made a good choice. Indeed full model is already heavy and complex, we don have to add another complexity.